README.md

```diff
@@ -45,7 +45,7 @@ mailing list [lightning-dev@lists.linuxfoundation.org][ml2].
 ## Getting Started
 
 c-lightning currently only works on Linux (and possibly Mac OS with some
-tweaking), and requires a locally running `bitcoind` (version 0.15 or
+tweaking), and requires a locally (or remotely) running `bitcoind` (version 0.15 or
```

Comment:
nit: Parentheses are not needed here. But then they are not needed in the above "and possibly Mac OS..." line either. This is a very good addition though.